### DISCUSSION TITLE
Adding Kenans sound pack to base game?

### DISCUSSION THREAD

**Discussion by thelonestander** [cataclysmbn/Cataclysm-BN#2105] [General]
Should we allow mainline BN to ship with a sound pack? I think so, as it makes the game more complete. Id like to get other peoples ideas here too on this. As I think it would be nice to have. A full game with mods, and graphics and ofc sounds. What do you guys think?

**Comment by yay855** [comment]
Honestly, it's a good idea. Cataclysm can be daunting to new players, and having a built-in sound pack will help reduce that at least some.

**Comment by olanti-p** [comment]
Definitely needs to be optional (recent release of Kenan's soundpack is half a gigabyte), but is doable and could totally be done for stable releases. Not sure about "experimental" releases, redownloading half a gig after even a tiny change sounds silly (but maybe ppl who don't use a launcher will find in convenient, dunno).

DDA have a solution that with some modifications should work with BN:
https://github.com/CleverRaven/Cataclysm-DDA/pull/57138
